src/main/webapp/app/exercises/shared/feedback/group/programming-feedback-groups.ts

```diff
@@ -22,8 +22,8 @@ class ProgrammingFeedbackGroupWrong extends FeedbackGroup {
     }
 
     shouldContain(feedbackItem: FeedbackItem): boolean {
-        const isReviewerFeedback = feedbackItem.type === 'Reviewer' && feedbackItem.credits && feedbackItem.credits < 0;
-        const isTestFeedback = feedbackItem.type === 'Test' && feedbackItem.credits !== undefined && feedbackItem.credits <= 0;
+        const isReviewerFeedback = feedbackItem.type === 'Reviewer' && feedbackItem.credits !== undefined && feedbackItem.credits < 0;
+        const isTestFeedback = feedbackItem.type === 'Test' && feedbackItem.positive !== undefined && !feedbackItem.positive;
```

Comment:
Would it be possible to compare feedbackItem.positive with `false`? This would be more understandable to me

Reply:
Would be possible but I really dislike comparing booleans

Reply:
okay